acceptance-tests/features/sign-in.feature

```diff
@@ -42,6 +42,47 @@ Feature: Users can sign in to the self-service experience
       And they see the toggle link "Show" on the field "password"
       Then they can not see the content in the password field
 
+  Rule: The user tries to sign in with an expired or invalid SMS Code
```

Comment:
Rename to : The user tries to submit SMS security code

Reply:
Thanks Boris, change made as requested.